packages/wasm-crypto/Cargo.toml

```
@@ -22,6 +22,7 @@ hmac = "0.7.0"
 # libsecp256k1 = "0.2.2"
 merlin = { version = "2.0", default-features = false }
 pbkdf2 = { version = "0.3.0", default-features = false }
+curve25519-dalek = { version = "2.0", default-features = false }
```

```suggestion
curve25519-dalek = { version = "2.1.0", default-features = false }
```

(As already available in the Cargo.lock. Cargo doesn't de-dupe nicely, so aligning versions is quite critical to keep down bloat)